keyboards/hazel/chaz/rp2040ce/info.json

```diff
@@ -0,0 +1,65 @@
+{
+    "manufacturer": "Hazel",
+    "keyboard_name": "Chaz",
+    "maintainer": "jasonhazel",
+    "url": "https://github.com/hazels-garage/chaz",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "nkro": false
+    },
+    "matrix_pins": {
+        "cols": ["GP9", "GP8", "GP7", "GP6", "GP5"],
+        "rows": ["GP0", "GP2", "GP4", "GP28", "GP1", "GP3", "GP27", "GP29"]
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x4A48",
+        "pid": "0x435A",
```

Comment:
`vid` and `pid` are identical for both `rp2040ce` and `promicro` directories, would you be able to move these two lines to `info.json` in the directory above (keyboard level)?